pulp_rpm/app/serializers/distribution.py

```diff
@@ -40,7 +40,7 @@ class ChecksumSerializer(serializers.ModelSerializer):
     """
 
     path = serializers.CharField(help_text=_("File path."))
-    checksum = serializers.CharField(help_text=_("Checksum for the file."))
+    checksum = serializers.CharField(help_text=_("Checksum for the file."), allow_null=True)
```

Comment:
I understand that this change is to match the model.
However, does anyone know the case when checksum can be null?
This field is a part of uniqueness constraint, every null in db is unique, so basically we can have duplicates in the database.

@fao89 , @dkliban , @daviddavis , @pavelpicka , any thoughts?

Reply:
If understand well https://release-engineering.github.io/productmd/treeinfo-1.0.html there is no case to checksum can be null.

Reply:
You are right @goosemania 
I reverted this change and I believe we need to change the Checksum model, I propose to do this migration in another issue/PR

Reply:
Thanks, I'll  open an issue

Reply:
https://pulp.plan.io/issues/7414